scripts/backend_test_shards.json

```diff
@@ -51,7 +51,8 @@
         "core.controllers.voice_artist_test",
         "core.jobs.batch_jobs.skill_migration_jobs_test",
         "core.jobs.batch_jobs.story_migration_jobs_test",
-        "core.jobs.batch_jobs.email_deletion_jobs_test"
+        "core.jobs.batch_jobs.email_deletion_jobs_test",
+        "scripts.install_python_dev_dependencies_test"
```

Comment:
Can we add this test to shard 2 as well (for load-balancing purposes)?

Reply:
Shard 1 is still running in under 30 mins, so I think we're currently satisfying the load-balancing requirements. I'm also not sure by what you mean by adding it to shard 2 "as well" since a given test can only be in one shard at a time

Reply:
Actually, nvm. Done